x-pack/plugin/sql/sql-cli/src/main/java/org/elasticsearch/xpack/sql/cli/Cli.java

```diff
@@ -102,6 +103,11 @@ public Cli(CliTerminal cliTerminal) {
             .withRequiredArg()
             .ofType(Boolean.class)
             .defaultsTo(Boolean.parseBoolean(System.getProperty("cli.check", "true")));
+        this.lenientOption = parser.acceptsAll(
+            Arrays.asList("l", "lenient"),
+            "Lenient and return first value for fields with multiple values"
```

Comment:
The description is a bit confusing. How about `Enable fields multi-value leniency. Returns the first value from the values array with no guarantee of consistent results. Accepts 'true' or 'false' values.`

Reply:
Yes, much better. Fixing it